index.html

```diff
@@ -0,0 +1,144 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+</head>
+<body>
+
+  <!-- HEADER SECTION -->
+  <div class="navbar">
+    <img class="item-1" src="./assets/wolox_logo.svg"/>
```

Comment:
img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.  add that prop to all the images

Reply:
pushed the changes